test/unit/attributes.js

```diff
@@ -4,127 +4,306 @@ import helpers from '../lib/helpers';
 import skate from '../../src/skate';
 
 describe('Attribute listeners', function () {
-  it('should listen to changes in specified attributes', function (done) {
+  'use strict';
+
+  describe('default values', function () {
+    it('should set a default value using the "default" option', function () {
+      var MyEl = skate('my-el', {
+        attributes: {
+          test: {
+            default: 'true'
+          }
+        }
+      });
+
+      new MyEl().getAttribute('test').should.equal('true');
+    });
+
+    it('should allow a callback to return a default value', function () {
+      var MyEl = skate('my-el', {
+        attributes: {
+          test: {
+            default: function () {
+              return 'true';
+            }
+          }
+        }
+      });
+
+      new MyEl().getAttribute('test').should.equal('true');
+    });
+
+    it('should not override if already specified', function () {
+      var MyEl = skate('my-el', {
+        attributes: {
+          test: {
+            default: 'true'
+          }
+        }
+      });
+
+      skate.init(helpers.fixture('<my-el test="false"></my-el>'))
+      .firstChild
+      .test
+      .should
+      .equal('false');
```

Comment:
No two space continuation indent? Looks like http://editorconfig.org/#file-format-details doesn't have that option.